README.md

```diff
@@ -1,2 +1,3 @@
 # devsecops-iam-roles
```

Comment:
This could be expanded

Reply:
Heh.

Reply:
Especially given @Vermyndax JUST brought up needing to have better user documentation 😛